key2pdf.js

```diff
@@ -21,7 +22,7 @@ var parse = require('date-fns/parse');  //https://github.com/date-fns/date-fns
 var format = require('date-fns/format');  //https://github.com/date-fns/date-fns
 var differenceInMilliseconds = require('date-fns/difference_in_milliseconds'); //https://github.com/date-fns/date-fns
 var jobs = [];
-
+var gDriveUpload = require('./gdrive');
 
```

Comment:
@bryancross The `.js` and `.json` extensions aren't explicitly required since node will [attempt those extensions by default](https://nodejs.org/api/modules.html#modules_file_modules). Just noticing we're not being consistent. Do you have a preference?